presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java

```diff
@@ -2556,6 +2557,40 @@ public void testViewCreation()
         }
     }
 
+    @Test
+    public void testHiveViewTranslationError()
+    {
+        String tableName = "test_hive_view";
+        ConnectorSession session = newSession();
+        Table table = Table.builder()
```

Comment:
We shouldn't need to create a new Hive view here, as these tests already create `presto_test_view` (referenced by the `view` field in this class). It should be enough to call
```java
metadata.getView(session, view);
```